### PR TITLE
update .aicoe-ci.yaml to include repo image in MOCJH

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies

#40 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Updates the .aicoe-ci.yaml file according to [these docs](https://github.com/AICoE/aicoe-ci#configuring-build-requirements) so that we can have new images built and pushed to quay based on this repo, and in turn made available on MOC JH.

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -6,11 +6,11 @@ check:
...
-#   build-stratergy: Source
-#   build_source_script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+build:
+  build-stratergy: Source
+  build_source_script: "image:///opt/app-root/builder"
```
Comment (4n4nd) [inline .aicoe-ci.yaml:11]: can you change this to `build-source-script`?

**Review by 4n4nd — CHANGES_REQUESTED** [review]
One small change requested, otherwise LGTM :+1:

File: .aicoe-ci.yaml (hunk 1 above)
Comment (4n4nd) [reply .aicoe-ci.yaml:11]: this was updated in the [template repo](https://github.com/aicoe-aiops/project-template/blob/master/.aicoe-ci.yaml#L11) later

Comment (MichaelClifford) [reply .aicoe-ci.yaml]: thanks @4n4nd, change made

File: .aicoe-ci.yaml [hunk 2]
```diff
@@ -6,11 +6,11 @@ check:
   # - thoth-build
```
Comment (4n4nd) [inline .aicoe-ci.yaml:6]: Oh also can you uncomment this line.
Sorry forgot to mention it before

**Review by 4n4nd — CHANGES_REQUESTED** [review]
my bad, forgot to mention this change

File: .aicoe-ci.yaml (hunk 2 above)
Comment (MichaelClifford) [reply .aicoe-ci.yaml]: :+1:  no worries, done.

**Review by 4n4nd — APPROVED** [review]
:+1:

**Comment by 4n4nd** [comment]
@MichaelClifford I think you can tag a new release now

**Pull request merged**